src/components/header/header.js

```diff
@@ -7,26 +7,32 @@ class Header extends HTMLElement {
     this.links = [
       {
         name: 'Home',
-        href: 'home.html',
-        active: this.checkPathname('index.html'),
+        href: 'home/',
+        active: this.checkPathname('home'),
       },
       {
         name: 'Films',
-        href: 'films.html',
-        active: this.checkPathname('films.html'),
+        href: 'pages/films/',
+        active: this.checkPathname('films'),
       },
       {
         name: 'Collections',
-        href: 'collections.html',
-        active: this.checkPathname('collections.html'),
+        href: 'pages/collection-details/',
+        active: this.checkPathname('collection-details'),
       },
     ];
   }
 
   // eslint-disable-next-line class-methods-use-this
   checkPathname(path) {
     const pathname = window.location.pathname.split('/');
-    return pathname[1] === path;
+    return pathname[pathname.length - 2] === path;
+    // If we delete with last '/' from adres then should be 'pathname.length - 1'
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  hrefJoin(href) {
+    return window.location.href.slice(0, window.location.href.search('Web-Development-Basics') + 23) + href
```

Comment:
+ 23? o co chodzi?

Reply:
nie rób tego 23 na sztywno bo nie wiadomo o co chodzi

Reply:
ok, faktycznie słabo to wygląda, poprawiłem, teraz jest czytelne